src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs

```diff
@@ -56,6 +56,7 @@ internal sealed class SolutionRestoreJob : ISolutionRestoreJob
 
         private NuGetOperationStatus _status;
         private int _packageCount;
+        private int _packageOnlyCount;
```

Comment:
`_packageCount` actually includes `csproj` references too.
https://github.com/NuGet/NuGet.Client/blob/048c1f435952666c470d85724b3d4ea1e100c168/src/NuGet.Clients/NuGet.SolutionRestoreManager/SolutionRestoreJob.cs#L448

Reply:
The link doesn't work for me, so I'm not sure what comment you're referring to.

Reply:
Sorry, here is new one. https://github.com/NuGet/NuGet.Client/pull/4228/files#r699562327

Reply:
Sorry, here is new link: https://github.com/NuGet/NuGet.Client/pull/4228/files#r699570325